docs/capabilities.md

```diff
@@ -0,0 +1,78 @@
+# Capabilities of an user required for a ceph-csi in an Ceph cluster
+
+Ceph uses the term capabilities to describe authorizing an authenticated user
+to exercise the functionality of the monitors, OSDs and metadata servers.
+Capabilities can also restrict access to data within a pool or pool namespace.
+A Ceph administrative user sets a user's capabilities when creating or
+updating a user.
+
+## RBD
+
+We have provisioner, controller expand and node stage secrets in storage class.
+For the provisioner and controller expand stage secret in storageclass, the
+user needs to have the  below mentioned ceph capabilities.
+
+```text
+"mon", "profile rbd",
+"mgr", "allow rw",
+"osd", "profile rbd"
+```
+
+And for the node stage secret in storageclass, the user needs to have the
```

Comment:
I am not sure what `node stage` refers to here, it is not very clear.
The "Node Host" runs the "Node Plugin" and does a Stage operation. The text now seems to try to express all terms into a single phrase. It would be better to explain what operations are done on the node (Stage, Publish, Unpublish, Unstage) and what capabilities these require (as Madhu indicates below).
The same cane be done for the operations executed by the provisioner.

Reply:
This may need to rephrase we have `node-stage-secret-name` storage class it is a secret name in secret we have userid and user key which is corresponds to the userid and his credentials in ceph cluster to perform some actions.

Reply:
okay, I will rephrase it.

Reply:
@nixpanic I dont feel like we need to explain about the operations that are done on the node in this document as this document mainly focuses on the capabilities that a user requires. What do you say @Madhu-1 on this? Please, Let me know your suggestions and if you feel like we need this to be mention here, I will add it.

Reply:
Jus rephrasing is required here.

Reply:
okay. I have added this thing in the beginning of the doc, So that it will be clear for the readers about the userid and key